src/mozillavpn.cpp

```diff
@@ -920,6 +920,7 @@ void MozillaVPN::reset(bool forceInitialState) {
   logger.debug() << "Cleaning up all";
 
   TaskScheduler::deleteTasks();
+  m_private->m_closeEventHandler.removeAllStackViews();
```

Comment:
This is needed because after a reset, we can still have stack views. We need to remove them all to go to the main view.